.github/workflows/tags_stable.yml

```diff
@@ -41,6 +41,8 @@ jobs:
         git diff HEAD
     - name: Create Pull Request
       uses: peter-evans/create-pull-request@v3
+      env:
+        GITHUB_TOKEN: ${{ secrets.ROBOT_CLICKHOUSE_COMMIT_TOKEN }}
```

Comment:
env won't do anything here, it should be [input](https://github.com/peter-evans/create-pull-request#action-inputs)

As well, it makes sense to add `do not test` to labels of created PRs